src/verify_getter.rs

```diff
@@ -61,15 +66,76 @@ fn verify_get_regular(attr_set: &SyntaxNode) -> Result<SyntaxNodeAndWhitespace>
     Ok(SyntaxNodeAndWhitespace { whitespace, node: deps_list})
 }
 
+fn find_or_insert_key_value_with_key(node: &SyntaxNode, key: &str, if_missing_template: SyntaxNode) -> Option<SyntaxNodeAndWhitespace> {
+    let found = find_key_value_with_key(&node, key);
+    if found.is_some() {
+        return found;
+    }
+    let count = node.children().count() + 2;
+
+    node.splice_children(count..count, vec![
+        rnix::NodeOrToken::Node(rnix::Root::parse("\n  ").syntax().clone_for_update()),
+        rnix::NodeOrToken::Node(if_missing_template),
+    ]);
+
+    let result = find_key_value_with_key(&node, key);
+    result
+}
+
+fn template_deps() -> SyntaxNode {
+    let python_env_template = r#"{
+  deps = [];
+}"#;
+    let ast = rnix::Root::parse(python_env_template);
+    let errors = ast.errors();
+    if errors.len() > 0 {
+        panic!("add_syntax_node had error: {:#?}", errors)
+    }
+    ast.syntax().first_child().unwrap().first_child().unwrap().clone_for_update()
+}
+
+fn template_env() -> SyntaxNode {
+    let python_env_template = r#"{
+  env = {
+    PYTHON_LD_LIBRARY_PATH = pkgs.lib.makeLibraryPath [];
+  };
+}"#;
+    let ast = rnix::Root::parse(python_env_template);
+    let errors = ast.errors();
+    if errors.len() > 0 {
+        panic!("add_syntax_node had error: {:#?}", errors)
+    }
+    ast.syntax().first_child().unwrap().first_child().unwrap().clone_for_update()
+}
+
+fn template_python() -> SyntaxNode {
+    let python_env_template = r#"{
+    PYTHON_LD_LIBRARY_PATH = pkgs.lib.makeLibraryPath [];
+}"#;
+    let ast = rnix::Root::parse(python_env_template);
+    let errors = ast.errors();
+    if errors.len() > 0 {
+        panic!("add_syntax_node had error: {:#?}", errors)
+    }
+    ast.syntax().first_child().unwrap().first_child().unwrap().clone_for_update()
+}
+
 fn verify_get_python(attr_set: &SyntaxNode) -> Result<SyntaxNodeAndWhitespace> {
-    let env = find_key_value_with_key(&attr_set, "env").context("expected to have an env key")?.node;
+    let env = find_or_insert_key_value_with_key(
+        &attr_set,
+        "env",
+        template_env())
```

Comment:
can we follow js conventions for multiline function calls? lisp conventions are harder to read :/

```suggestion
    let env = find_or_insert_key_value_with_key(
        &attr_set,
        "env",
        template_env(),
    )
```

same applies below